src/include/rados/librados.h

```diff
@@ -15,6 +15,10 @@
 #ifndef CEPH_LIBRADOS_H
 #define CEPH_LIBRADOS_H
 
+#ifdef _WIN32
+#include "../windows/windefs.h"
```

Comment:
After a second thought,  we should avoid including this here. It's sole purpose is to avoid ``boost`` include order issues, for which reason I've moved it to the Cmake file.